### DISCUSSION TITLE
Loading library after the map was created

### DISCUSSION THREAD

**Discussion by mcepo** [geoman-io/leaflet-geoman#989] [Questions & Help] [answered]
Is it possible to import leaflet-geoman after the map was already created ?

I posted a question regarding this on [stack-overflow](https://stackoverflow.com/questions/69224411/webpack-lazy-loading-leaflet-geoman)

**Comment by Falke-Design** [comment, marked as answer]
Geoman is adding it self to the map while initializing the map. This means if L.Map is called before Geoman-Script is added to your site, Geoman will not be added to the map. 
Then you need to call `L.PM.reInitLayer(map)`.

**Reply by mcepo** [reply]
This worked, thx.
I assume lazy loading this library, ie after the map was created, is a common use case, so it might be worthwhile adding a section in the documentation about this.

**Reply by Falke-Design** [reply]
Yes you are right. If you want, a PR is welcome
